dicom_util/src/main/java/com/google/cloud/healthcare/Util.java

```diff
@@ -0,0 +1,10 @@
+package com.google.cloud.healthcare;
```

Comment:
I'm not sure it's worth introducing a new class for just this one method. I think duplicating it is fine given it's just one line.

Reply:
I just really dislike code duplication. There could be more clients later too (well, there were more, during testing/working it out phase).

Reply:
Ok, can we rename this class to StringUtil instead?

Reply:
Ok